File: code/1009/1009.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const base = parseFloat(lines[1]);
const sales = parseFloat(lines[2]);

const newSalary = base + (sales * 0.15);

console.log(`TOTAL = R\$ ${newSalary.toFixed(2)}`);

File: code/1009/1009.cpp
#include <iomanip>
#include <iostream>
#include <string>

using namespace std;

int main() {

  string name;
  double base;
  double sales;
  double newSalary;

  cin >> name >> base >> sales;

  newSalary = base + (sales * 0.15);

  cout << "TOTAL = R$ " << fixed << setprecision(2) << newSalary << endl;

  return 0;
}
